### PR TITLE
[containerd]Fixing container commands when mode is local and state is disabled

### PR DESCRIPTION
**Pull request by dgsudharsan**
Signed-off-by: Sudharsan Dhamal Gopalarathnam <sudharsand@nvidia.com>

#### Why I did it
During warm-reboot and fast-reboot the below error logs appear
Feb  3 22:05:15.187408 r-lionfish-13 ERR container: docker cmd: kill for nat failed with 404 Client Error for http+docker://localhost/v1.41/containers/nat/json: Not Found ("No such container: nat")

The container command when called for local mode doesn't check if it is enabled before calling docker kill which throws the above errors. 
https://github.com/Azure/sonic-utilities/blob/b6ca76b4821453171d9607383b808385968eeeeb/scripts/fast-reboot#L699

#### How I did it
Checking feature state if local mode and returning error exit code along with valid debug message.

#### How to verify it
Added UT to verify it.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: src/sonic-ctrmgrd/ctrmgr/container [hunk 1]
```diff
@@ -301,19 +311,27 @@ def container_kill(feature, **kwargs):
...
-    set_owner, _ = read_config(feature)
+    ret = SUCCESS
+    set_owner, _ , state = read_config(feature)
     current_owner, remote_state, _ = read_state(feature)
     docker_id = container_id(feature)
     remove_label = (set_owner != "local") or (current_owner != "local")
 
-    debug_msg("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{}".format(
-        feature, set_owner, current_owner, remote_state, docker_id))
+    print("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{} state:{}".format(
```
Comment (stepanblyschak) [inline src/sonic-ctrmgrd/ctrmgr/container:320]: Change back to debug_msg?

File: src/sonic-ctrmgrd/ctrmgr/container [hunk 2]
```diff
@@ -301,19 +311,27 @@ def container_kill(feature, **kwargs):
...
+        feature, set_owner, current_owner, remote_state, docker_id, state))
 
+    print(remove_label)
```
Comment (stepanblyschak) [inline src/sonic-ctrmgrd/ctrmgr/container:323]: Maybe print is left after some debugging?

File: src/sonic-ctrmgrd/ctrmgr/container [hunk 3]
```diff
@@ -301,19 +311,27 @@ def container_kill(feature, **kwargs):
...
     if remove_label:
         set_label(feature, False)
 
+    if set_owner == "local":
+        print("returning failure")
```
Comment (stepanblyschak) [inline src/sonic-ctrmgrd/ctrmgr/container:328]: Remove print

File: src/sonic-ctrmgrd/ctrmgr/container (hunk 1 above)
Comment (dgsudharsan) [reply src/sonic-ctrmgrd/ctrmgr/container]: Done

File: src/sonic-ctrmgrd/ctrmgr/container (hunk 2 above)
Comment (dgsudharsan) [reply src/sonic-ctrmgrd/ctrmgr/container]: Done

**Review by stepanblyschak — APPROVED** [review]

**Pull request closed without merging**